service/resource/legacy/error.go

```diff
@@ -1,24 +0,0 @@
-package legacy
```

Comment:
The legacy resource is also removed because the two new resources do the do now.